src/c++/perf_analyzer/inference_profiler.cc

```diff
@@ -580,6 +583,8 @@ InferenceProfiler::ProfileHelper(
   all_timestamps_.clear();
   previous_window_end_ns_ = 0;
 
+  triton_metrics_manager_->StartQueryingTritonMetrics();
```

Comment:
Instead of doing this, I would propose to call this function inside Measure and attach the results to `status_summary`. Right now, there is no way to distinguish the metrics collected inside stable trials from unstable trials.

Reply:
Discussed offline and I'll move the querying logic within Measure() 👍

Reply:
Changed to store results in status summary 👍